src/containers/hashmap.d

```diff
@@ -85,12 +85,26 @@ struct HashMap(K, V, Allocator = Mallocator, alias hashFunction = generateHash!K
 		}
 	}
 
-	~this()
+	~this() nothrow
 	{
-		import stdx.allocator : dispose;
+		try
+			clear();
+		catch (Throwable)
+			assert(false);
+	}
+
+	/**
+	 * Removes all items from the map
+	 */
+	void clear()
+	{
+		import std.experimental.allocator : dispose;
```

Comment:
What's the reason for not using stdx.allocator? The experimental package may break at any time again :/

Reply:
That's something left over from when I was working on getting this working. Maybe the allocators change broke something else downstream, but the containers library itself seemed to work fine.